Solutions/DigitalGuardianDLP/Analytic Rules/DigitalGuardianFilesSentToExternalDomain.yaml

```diff
@@ -0,0 +1,37 @@
+id: 5f75a873-b524-4ba5-a3b8-2c20db517148
+name: Digital Guardian - Bulk exfiltration to external domain
+description: |
+  'Detects bulk exfiltration to external domain.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: DigitalGuardianDLP
+    dataTypes:
+      - DigitalGuardianDLPEvent
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Exfiltration
+relevantTechniques:
+  - T1048
+query: |
+  let threshold = 10;
+  let corp_domain = dynamic(['example.com']);

```

Comment:
Please add a comment to let users know to add their own domains.

Reply:
added comment